src/components/layouts/BlankLayout.tsx

```diff
@@ -0,0 +1,7 @@
+import { PropsWithChildren } from "react";
+
+type Props = PropsWithChildren<{}>;
```

Comment:
Do you have rules for using interfaces or types?
https://github.com/monstar-lab-oss/admin-panel-template-reactjs/discussions/88#discussioncomment-942054

Reply:
I'm not particular about it, but I prefer Props because it seems to be better at discriminated union type 🤔
https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/20

Reply:
Seems this discussion haven't been active... 😓  I'll resolve this comment since it should be discussed in https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/20